console/backend/src/main/java/org/frankframework/management/web/FrankApiBase.java

```diff
@@ -15,68 +15,44 @@
 */
 package org.frankframework.management.web;
 
-import java.security.Principal;
-
-import jakarta.servlet.ServletConfig;
-import jakarta.servlet.http.HttpServletRequest;
-import jakarta.ws.rs.core.Context;
-import jakarta.ws.rs.core.EntityTag;
-import jakarta.ws.rs.core.Request;
-import jakarta.ws.rs.core.Response;
-import jakarta.ws.rs.core.Response.ResponseBuilder;
-import jakarta.ws.rs.core.SecurityContext;
-import jakarta.ws.rs.core.UriInfo;
-
-import jakarta.annotation.Nonnull;
-import org.apache.commons.lang3.StringUtils;
-import org.apache.cxf.jaxrs.JAXRSServiceFactoryBean;
-import org.apache.cxf.jaxrs.spring.JAXRSServerFactoryBeanDefinitionParser.SpringJAXRSServerFactoryBean;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
+import org.frankframework.management.bus.OutboundGateway;
+import org.frankframework.management.web.configuration.DeprecationInterceptor;
+import org.frankframework.util.ResponseUtils;
 import org.springframework.beans.BeansException;
 import org.springframework.beans.factory.InitializingBean;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.context.ApplicationContext;
 import org.springframework.context.ApplicationContextAware;
 import org.springframework.core.env.Environment;
+import org.springframework.http.ResponseEntity;
 import org.springframework.messaging.Message;
 
+import jakarta.servlet.http.HttpServletRequest;
 import lombok.Getter;
-
-import org.frankframework.management.bus.OutboundGateway;
-
-import org.frankframework.util.ResponseUtils;
-import org.frankframework.web.filters.DeprecationFilter;
-
-/**
- * Base class for API endpoints.
- * Contains helper methods to read JAX-RS multiparts and handle message conversions to JAX-RS Responses.
- * @author	Niels Meijer
- */
+import lombok.NonNull;
 
 public abstract class FrankApiBase implements ApplicationContextAware, InitializingBean {
 
-	@Context protected ServletConfig servletConfig;
-	@Context protected @Getter SecurityContext securityContext;
-	@Context protected @Getter HttpServletRequest servletRequest;
+	@Autowired
+	protected @Getter HttpServletRequest servletRequest;
+
 	private @Getter ApplicationContext applicationContext;
-	@Context protected @Getter UriInfo uriInfo;
-	@Context private Request rsRequest;
 	private @Getter Environment environment;
 
-	private JAXRSServiceFactoryBean serviceFactory = null;
-
 	protected Logger log = LogManager.getLogger(this);
 
 	protected final OutboundGateway getGateway() {
 		return getApplicationContext().getBean("outboundGateway", OutboundGateway.class);
 	}
 
-	@Nonnull
+	@NonNull
```

Comment:
We don't want to use the Lombok `@NonNull`, but the Jakarta version, which was the right one before.

Reply:
Ik zie er ook org.springframework.lang.Nullable en org.springframework.lang.NonNull tussen staan :)